agent/Makefile

```diff
@@ -241,48 +253,42 @@ install-python3-setup: install-util-scripts install-lib
 	${COPY} requirements.txt ${DESTDIR}
 	mkdir -p ${DESTDIR}/python3
 	(cd ..; SKIP_GENERATE_AUTHORS=1 SKIP_WRITE_GIT_CHANGELOG=1 python3 setup.py install --prefix=${DESTDIR}/python3)
-	${COPY} ${DESTDIR}/python3/bin/pbench-config ${UTILDIR}/
-	${COPY} ${DESTDIR}/python3/bin/pbench-clear-tools ${UTILDIR}/
-	${COPY} ${DESTDIR}/python3/bin/pbench-list-triggers ${UTILDIR}/
-	${COPY} ${DESTDIR}/python3/bin/pbench-clear-results ${UTILDIR}/
-	${COPY} ${DESTDIR}/python3/bin/pbench-cleanup ${UTILDIR}/
-	${COPY} ${DESTDIR}/python3/bin/pbench-list-tools ${UTILDIR}/
-	${COPY} ${DESTDIR}/python3/bin/pbench-register-tool-trigger ${UTILDIR}/
+	${COPY} $(addprefix ${DESTDIR}/python3/bin/, ${click-scripts}) ${UTILDIR}/
```

Comment:
Well that's a neat trick.

Reply:
Yeah, `make` has all sorts of cool widgets.